Makefile

```diff
@@ -39,6 +42,19 @@ test-filter:
 build-integration-test:
 	@zig build integration_test
 
+
+# Task for running the custom integration process
+run-custom-integration-test:
```

Comment:
just for the sake of demonstration i created a separate entry point to run the integration tests that 

- specifically look up all the cairo files matching an arbitary glob
- write all matches to a text file
- compile all matches
- pass the text file into the integration_test runner to actually run